_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -0,0 +1,316 @@
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			<span lang="fr">The Dutch phrase</span> "Hij ging met de kippen op stok"
+			<span lang="fr"
+				>literally translates into "He went to roost with the chickens", but it means that he went to bed early.</span
+			>
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+This `div` element has a `lang` attribute value of `fr` (French), which does not match its [most common language][]. The most common language is English because the accessible texts are English, and all other text is in a `p` element with a `lang` attribute value of `nl`, which also doesn't match its common language.
+
+```html
+<html lang="fr">
+	<head>
+		<title>Feu d'artifice du nouvel an</title>
+	</head>
+	<body>
+		<div lang="fr">
+			<img src="/test-assets/shared/fireworks.jpg" alt="Fireworks over Paris" />
+			<p lang="nl">
+				Bonne année !
+			</p>
+		</div>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+This `div` element has a `lang` attribute value of `fr` (French), which does not match its [most common language][]. The most common language is English because the accessible name of the `img` element is English. The `lang` attribute on the `p` element is effectively ignored. The `p` element is not applicable because it has no [text with the same programmatic language][] since its content is neither [visible][] nor [included in the accessibility tree][].
+
+```html
+<html lang="fr">
+	<head>
+		<title>Feu d'artifice du nouvel an</title>
+	</head>
+	<body>
+		<div lang="fr">
+			<img src="/test-assets/shared/fireworks.jpg" aria-labelledby="caption" />
+			<p lang="en" id="caption" hidden>
+				Fireworks over Paris
+			</p>
+		</div>
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+There are no HTML elements in this document.
+
+```svg
+<svg xmlns="http://www.w3.org/2000/svg" lang="en">
+    <text x="0" y="0">I love ACT rules!</text>
+</svg>
+```
+
+#### Inapplicable Example 2
+
+There is no descendant of a `body` element with a `lang` attribute.
+
+```html
+<html lang="en">
+	<body>
+		<p>I love ACT rules!</p>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+This `p` element has an invalid language tag.
+
+```html
+<html lang="en">
+	<body>
+		<p lang="français">
+			I love ACT rules!
+		</p>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+The first `p` element has a no [text with the same programmatic language][] because it has no content.
+
+```html
+<html lang="en">
+	<body>
+		<p lang="fr"></p>
+		<p>I love ACT rules!</p>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 5
+
+This `p` element has a no [text with the same programmatic language][] because it has no content that is either [visible][] or [included in the accessibility tree][].
```

Comment:
```suggestion
This `p` element has no [text with the same programmatic language][] because it has no content that is either [visible][] or [included in the accessibility tree][].
```